tests/test_storage_interface.py

```diff
@@ -141,6 +141,15 @@ def test_replace_only_existing(self):
 
         self.fs.replace(file_id, FILE_CONTENT)
 
+    @raises(IOError)
+    def test_create_set_id(self):
+        file_id = self.fs.create(FILE_CONTENT, 'file.txt')
+        assert self.fs.exists(file_id)
+        self.fs.delete(file_id)
+        assert not self.fs.exists(file_id)
+
+        self.fs.create(FILE_CONTENT, fileid=file_id)
```

Comment:
I think you want a `assert self.fs.exists(file_id)` affert this line. Otherwise we wouldn't be testing much.

Reply:
Added.